package.nls.json

```diff
@@ -1,5 +1,9 @@
 {
   "add.eventListener.breakpoint": "Toggle Event Listener Breakpoints",
+  "add.xhr.breakpoint": "Add XHR/fetch Breakpoint",
+  "breakpoint.xhr.contains":"Break when URL contains:",
+  "breakpoint.xhr.any":"Any XHR/fetch",
+  "edit.xhr.breakpoint": "Edit XHR/fetch Breakpoint",
```

Comment:
Wonder if just calling them "Fetch Breakpoints" would confuse people. I'm not sure anyone who's learned JavaScript in the last 5 years has ever written an XHR request manually 😛

Reply:
I have seen people use it, and since it is still apart of the Javascript standard, and that Chromium has that labeled as "XHR/Fetch", I labeled it as such. I personally feel like it would confuse people, but that is just me.